src/app/components/Navigation/Navigation.module.css

```diff
@@ -0,0 +1,34 @@
+.navigation {
+  position: fixed;
+  bottom: 0;
+  left: 0;
```

Comment:
I think it's better to control the layout from the potential parent component/element of this component. So we won't need those properties